health-services/egov-notification-push/start.sh

```
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if [[ -z "${JAVA_OPTS}" ]];then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, [[ ]] is undefined.

(SC3010)